src/helpers/useFetchAdvices.ts

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from "react";
+import { getAdvices } from "./getAdvices";
+import { Advice } from "../types/AdviceTypes";
+
+export const useFetchAdvices = () => {
+
+  const [adviceState, setAdviceState] = useState<Advice>();
+
+  const getTips = async () => {
+    const newAdvice = await getAdvices();
+    setAdviceState(newAdvice);
+  }
+
+  useEffect(() => {
+    getTips();
+  },[]);
+  
+  return { adviceState, getTips }
```

Comment:
Veo que `getTips` no se usa, podrías quitarlo del return y en su lugar puedes añadir lo siguiente:

Tal vez el `adviceState` puede ser renombrado solo a `advice` ya que se sabe que es parte de un state y el añadirle `state` de nuevo podría ser redundante

`error`: Aquí cachas cualquier error que haya ocurrido a nivel servicio y puedes pintarlo en la interfaz para dar a conocer que algo sucedió al traer el "advice". Siempre que trabajes con llamadas asíncronas, es buena práctica cachar errores que puedan suceder. 🧷 

`loading`: También puedes crear un estado de tipo booleano para setear cuando la llamada asíncrona está en progreso y cuando ha terminado. Esto es opcional 🌾 

```suggestion
  return { advice: adviceState, error, loading }
```

Reply:
En cuanto a la función `getTips` lleva un botón que al dar clic, activa dicha función y muestra otro "advice" aleatoriamente.
Del resto los cambios han sido realizados. 🤙🏼